airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/ContainerOrchestratorApp.java

```diff
@@ -80,8 +83,16 @@ public ContainerOrchestratorApp(
 
   private void configureLogging() {
     for (final String envVar : OrchestratorConstants.ENV_VARS_TO_TRANSFER) {
+      LOGGER.info("checking env var: " + envVar);
       if (envMap.containsKey(envVar)) {
+        LOGGER.info("setting property env var: " + envVar);
         System.setProperty(envVar, envMap.get(envVar));
+        final String getEnvResult = System.getenv(envVar);
+        LOGGER.info("getting env " + envVar);
+        LOGGER.info(getEnvResult);
```

Comment:
Is it for debug only? This will need to be removed, we may have some credential in the env variables.